kf_lib_data_ingest/target_apis/kids_first.py

```diff
@@ -154,23 +154,19 @@
     },
     'genomic_file': {
         'standard_concept': CONCEPT.GENOMIC_FILE,
-        'links': {
-            'sequencing_experiment_id': CONCEPT.SEQUENCING.UNIQUE_KEY
-        },
         'properties': {
             "external_id": CONCEPT.GENOMIC_FILE.UNIQUE_KEY,
             "file_name": CONCEPT.GENOMIC_FILE.FILE_NAME,
-            "file_format": None,
-            "data_type": None,
-            "availability": None,
+            "file_format": CONCEPT.GENOMIC_FILE.FILE_FORMAT,
+            "data_type": CONCEPT.GENOMIC_FILE.DATA_TYPE,
+            "availability": CONCEPT.GENOMIC_FILE.AVAILABILITY,
             "controlled_access": None,
             "is_harmonized": CONCEPT.GENOMIC_FILE.HARMONIZED,
-            "paired_end": CONCEPT.READ_GROUP.PAIRED_END,
-            "hashes": CONCEPT.GENOMIC_FILE.UNIQUE_KEY,
-            "size": CONCEPT.GENOMIC_FILE.UNIQUE_KEY,
-            "urls": CONCEPT.GENOMIC_FILE.UNIQUE_KEY,
-            "acl": CONCEPT.GENOMIC_FILE.UNIQUE_KEY,
-            "reference_genome": CONCEPT.GENOMIC_FILE.UNIQUE_KEY,
+            "hashes": CONCEPT.GENOMIC_FILE.HASH,
+            "size": CONCEPT.GENOMIC_FILE.SIZE,
+            "urls": CONCEPT.GENOMIC_FILE.URL,
```

Comment:
doesn't urls need to be a list?

Reply:
Yes it does but we don't have a way to define types in the target schema yet. That would be kind of a big change and require some design + discussion. 

For right now, I propose we make a fix for https://github.com/kids-first/kf-lib-data-ingest/issues/313 and then the ingest pkg dev can map this column to a python list containing their URL like in [this example](https://github.com/kids-first/kf-ingest-packages/blob/26f8c1981278eb501196f50a840cbe03eb4155ac/kf_ingest_packages/common/extract_configs/s3_object_info.py#L127) and then the value will go directly into `urls` during the standard to target transformation.

Reply:
then change the concept name to URLS plural

Reply:
Ok, then in that case, what do you think about: `HASHES_DICT` and `URL_LIST`? Bc if we name them `HASHES` and `URLS` how will someone know one is meant to be a dict vs a list

Reply:
ok

Reply:
this is all hopefully temporary anyway until gen3 usage is fixed